src/main/java/com/spotify/docker/client/messages/swarm/Placement.java

```diff
@@ -41,10 +41,46 @@
   @JsonProperty("Constraints")
   public abstract ImmutableList<String> constraints();
 
+  @Nullable
+  @JsonProperty("Preferences")
+  public abstract ImmutableList<PlacementPreference> preferences();
```

Comment:
Can you add a javadoc stating when this is available? See https://github.com/spotify/docker-client/blob/master/src/main/java/com/spotify/docker/client/messages/HostConfig.java#L222-L226